docs/conf.py

```diff
@@ -106,3 +108,22 @@
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = []
+
+# Support external links to specific versions of the files in the Github repo
+branch = os.environ.get('READTHEDOCS_VERSION')
+if branch == None or branch == 'latest':
+    branch = 'master'
+
+scm_raw_web = 'https://raw.githubusercontent.com/mauriciovasquezbernal/opentelemetry-python/' + branch
```

Comment:
looks like these should be pointed at the main telemetry repo.

Reply:
Will update just before merging, otherwise I cannot see the results on my testing readthedocs instance.